.github/workflows/cpp-packaging.yml

```diff
@@ -417,19 +441,28 @@ jobs:
             sleep 300
           done
           set -e
-          cd firebase-cpp-sdk-final
-          unzip ../firebase_cpp_sdk.zip
-      - name: upload artifacts
+
+      - name: compute SDK hash
+        run: |
```

Comment:
Should we force the shell to "bash" here too since we are redirecting to a file since PowerShell redirection's default encoding is UTF-16 not UTF-8?

Reply:
It shouldn't matter as this step only runs on Ubuntu, but I forced it nonetheless.